src/category_theory/limits/shapes/zero.lean

```diff
@@ -212,12 +212,24 @@ def zero_morphisms_of_zero_object : has_zero_morphisms C :=
   comp_zero' := λ X Y Z f, by { dunfold has_zero.zero, rw ←category.assoc, congr, }}
 
 /-- A zero object is in particular initial. -/
-lemma has_initial : has_initial C :=
+def zero_is_initial : is_initial (0 : C) :=
+is_initial.of_unique 0
+/-- A zero object is in particular terminal. -/
+def zero_is_terminal : is_terminal (0 : C) :=
+is_terminal.of_unique 0
+
+/-- A zero object is in particular initial. -/
+@[priority 10]
+instance has_initial : has_initial C :=
 has_initial_of_unique 0
 /-- A zero object is in particular terminal. -/
-lemma has_terminal : has_terminal C :=
+@[priority 10]
+instance has_terminal : has_terminal C :=
 has_terminal_of_unique 0
 
+instance has_strict_initial : initial_mono_class C :=
```

Comment:
The linter is [complaining](https://github.com/leanprover-community/mathlib/runs/3062958306#step:6:14) about the priority of this instance.

Reply:
As it should! Fixed